public/js/map_direct.js

```diff
@@ -59,13 +59,13 @@ function submitEntry(obj, id) {
     let gasoline = Number(!($(".mapboxgl-popup-content").find('.gasoline img').hasClass('no-gas')));
     let diesel = Number(!($(".mapboxgl-popup-content").find('.diesel img').hasClass('no-gas')));
     let lpg = Number(!($(".mapboxgl-popup-content").find('.lpg img').hasClass('no-gas')));
-    validateCaptcha((token) => {
+    // validateCaptcha((token) => {
         let data = {
```

Comment:
Are we leaving IE users (among others) out on purpose? 

https://caniuse.com/#search=let